### DISCUSSION TITLE
Deploy Issue

### DISCUSSION THREAD

**Discussion by redblame315** [vercel/community#6483] [Bug Report]
### Page to Investigate

https://vercel.com/new/redblame315s-projects/import?s=https%3A%2F%2Fgithub.com%2Fredblame315%2Fqoves_header&hasTrialAvailable=1&showOptionalTeamCreation=false&project-name=qoves-header&framework=nextjs&totalProjects=1&remainingProjects=1

### Steps to Reproduce

Hello!
I hope you are doing well.

I have faced this error when deploy the next.js project.
Your Team encountered an unknown problem. Please reach out to our support team for details. [Learn More](https://vercel.com/help)

Look forward to your help with this.
Best Wishes.

**Comment by amyegan** [comment]
Thanks for reporting this, @redblame315. It should be fixed for you now. I responded in https://github.com/vercel/community/discussions/6027#discussioncomment-9095701 as well
